addons/im_livechat_mail_bot/models/mail_bot.py

```diff
@@ -11,13 +11,23 @@ def _get_answer(self, record, body, values, command):
         odoobot_state = self.env.user.odoobot_state
         if self._is_bot_in_private_channel(record):
             if odoobot_state == "onboarding_attachement" and values.get("attachment_ids"):
+                self.env["mail.canned.response"].create({
+                    "source": "Thanks",
+                    "substitution": "Thanks for your feedback. Goodbye!",
+                    "description": "This is a temporary canned response to see how canned responses work.",
```

Comment:
Shouldn't this be translated in the language of the user?

Reply:
Done in odoo/odoo#165336